milestones/exceptions.py

```diff
@@ -12,6 +12,10 @@ class InvalidContentKeyException(ValidationError):
     """ Course content/module/usage key validation exception class """
 
 
+class InvalidRequirementsException(ValidationError):
```

Comment:
I know it's a bit lengthy, but we should probably rename this to InvalidCourseContentMilestoneRequirementsException